pkg/util/util.go

```diff
@@ -1004,3 +1004,49 @@ func ValidateURL(sourceURL string) error {
 
 	return nil
 }
+
+// ValidateFile validates the file
+func ValidateFile(filePath string) error {
+	// Check if the file path exist
+	file, err := os.Stat(filePath)
+	if err != nil {
+		return err
+	}
+
+	if file.IsDir() {
+		return errors.Errorf("%s exists but it's not file", filePath)
+	}
+
+	return nil
+}
+
+// CopyFile copies file from source path to destination path
+func CopyFile(srcPath string, dstPath string) error {
```

Comment:
Duplicacy of the function `CopyFile`. We already have the same function in our code https://github.com/openshift/odo/blob/master/tests/helper/helper_filesystem.go#L144

Reply:
It is just a file copy paste. So without duplicating It, we can use the same function in integration test code for sure. Can you please update the integration test code accordingly

Reply:
Done